### ISSUE TITLE
Missing Schema Description

### ISSUE THREAD

**Issue by johannwagner** [OpenEnergyPlatform/oeplatform#526]
We needed the schema description for some more schemas, which are not listed within the wiki.
(https://wiki.openmod-initiative.org/wiki/Database_schema)

+ emission
+ openstreetmap

**Comment by jh-RLI** [comment]
According to the related Redmine [ticket](http://redmine.iks.cs.ovgu.de/oe/projects/szenariendb/issues?page=3&query_id=85) @han-f can contribute to this.

**Comment by jh-RLI** [comment]
She's not available right now. I added the description in the wiki.

**Comment by han-f** [comment]
Should I still do something here? I can review the descriptions on toep?

**Comment by MGlauer** [comment]
> Should I still do something here? I can review the descriptions on toep?

Yes, in my opinion the descriptions need some work, but this is not essential for the release of v0.6.0

**Comment by han-f** [comment]
OK, I can work on all of them at some point. Should I simply make suggestions here in this issue? That would be easiest for me as I can then take one by one when I have some time in between also.

**Comment by han-f** [comment]
suggestion for scenario:
scenario data in the broadest sense. Includes input and output data from models that project scenarios into the future. Example inputs: assumptions made about future developments of key parameters such as energy prices and GDP. Example outputs: projected electricity transmission, projected greenhouse gas emissions. Note that inputs to one model could be an output of another model and the other way around.

**Comment by han-f** [comment]
schema emission:
do we need this? I think this would be included in scenario as (mostly) output of models.

**Comment by han-f** [comment]
> Simply that they are collected and provided without the context of belonging to a specific scenario?
Yes, that is probably it.

**Comment by han-f** [comment]
suggestion for economy: 
data related to economic activities. Examples: sectoral value added, employment, sectoral inputs and outputs, GDP, prices of commodities.

**Comment by han-f** [comment]
suggestion for emission
data on emissions. Examples: total greenhouse gas emissions, CO2-emissions, energy-related CO2-emissions, methane emissions etc.

**Comment by han-f** [comment]
suggestion for model_draft:
unfinished data. Note: there is no version control and data is still volatile.

**Comment by han-f** [comment]
suggestion for society:
demographic data such as census data, emplyoment / unemployment, living conditions etc.

**Comment by han-f** [comment]
suggestion for policy
data on policies and measures. This could, for example, include a list of renewable energy policies per European Member State. It could also be a list of climate related policies and measures in a specific country.

**Comment by han-f** [comment]
suggestion for boundaries:
data that depicts boundaries, such as geographic, administrative or political boundaries. Such data comes as polygons.

**Comment by han-f** [comment]
suggestion for climate:
data related to climate and weather. This includes, for example, precipitation, temperature, cloud cover and atmospheric conditions.

**Comment by han-f** [comment]
suggestion for demand:
data on demand. Demand can relate to commoditie but also to services.

**Comment by han-f** [comment]
suggestion for supply:
data on supply. Supply can relate to commodities but also to services.

**Comment by han-f** [comment]
Writing these suggestions here I think many of the current schemas may have overlaps and could include the same things. Demand and supply for example could also be seen as inputs and outputs to economic sectors if viewed from an economic angle. And I also think the type of model data that the OEP is open to spans also beyond a very strict definition of energy system model.

**Comment by han-f** [comment]
@l-emele done for the moment. I think these sugestions are far from perfect, but they may give a better sense to the user what could be put or find where.

**Comment by l-emele** [comment]
Looks good. Just one minor suggestions for changes: 
> suggestion for emission:
> data on emissions. Examples: total greenhouse gas emissions, CO2-emissions, energy-related CO2-emissions, methane emissions, **air pollutants** etc.

**Comment by MGlauer** [comment]
@han-f Are those descriptions reads to be transfered?

**Comment by han-f** [comment]
From my point of view yes - we can still make them prettier in a future release, correct?

**Comment by MGlauer** [comment]
Yes, of course.

**Comment by han-f** [comment]
And some more suggestions, for the missing updates:

environment: data that has to do with the environment in the broadest sense. This could relate, for example, to mapping of environmental resources, land cover, protected areas etc.

**Comment by han-f** [comment]
grid: data that relates to infrastructure, such as power lines, pipelines etc.

**Comment by han-f** [comment]
> grid: data that relates to infrastructure, such as power lines, pipelines etc.

I think this does not necessarily only relate to energy infrastructure. Some models may make use of other infrastructure such as railways, roads etc. This could also fall under grid? Or would that be a net? Or would a better term for grid be net?

**Comment by han-f** [comment]
reference: data that serves as reference, such as literature, data sources, etc.

**Comment by han-f** [comment]
> reference: data that serves as reference, such as literature, data sources, etc.

@l-emele @wingechr: may this instead be the current target schema for our conversion and GWP tables? While writing the definition I thought that these actually are some kind of reference. If you agree, then I would rephrase the definition to: 

reference: data that serves as reference, such as literature, data sources, but also converters and lists of abbreviations etc.

**Comment by l-emele** [comment]
> > reference: data that serves as reference, such as literature, data sources, etc.
> 
> @l-emele @wingechr: may this instead be the current target schema for our conversion and GWP tables? While writing the definition I thought that these actually are some kind of reference. If you agree, then I would rephrase the definition to:
> 
> reference: data that serves as reference, such as literature, data sources, but also converters and lists of abbreviations etc.

My understanding is that the reference scheme only contains tables that are related to the abandoned literature collection. So it is not the place for converters and lists of abbreviations.

**Comment by christian-rli** [comment]
Is this issue still relevant or has it been addressed?
